paper-server/patches/sources/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -674,3 +674,25 @@
  
                  for (int i = 0; i < this.inventory.getContainerSize(); i++) {
                      ItemStack item = this.inventory.getItem(i);
+@@ -2089,12 +_,20 @@
+     }
+ 
+     public boolean hasClientLoaded() {
+-        return this.clientLoaded || this.clientLoadedTimeoutTimer <= 0;
++        return this.clientLoaded; // Paper - Add PlayerLoadedEvent
```

Comment:
This is used in another places.. this change not break any?

Reply:
No, the boolean is set to true. Take a look at the changes below.